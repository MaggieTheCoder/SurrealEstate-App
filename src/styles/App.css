@import url("https://fonts.googleapis.com/css2?family=Rowdies:wght@300&display=swap");

body {
  font-family: "Rowdies", cursive;
  margin: 0;
  padding: 0;
  background-color: #d3d3d3;
}
