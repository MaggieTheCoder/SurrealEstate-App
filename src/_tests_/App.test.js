import { render, screen } from "@testing-library/react";
import App from "../components/App";

test("renders learn react link", () => {
  render(<App />);
  const Element = screen.getByText(/Surreal Estate/i);
  expect(Element).toBeInTheDocument();
});
