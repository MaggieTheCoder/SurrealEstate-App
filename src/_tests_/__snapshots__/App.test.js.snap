// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <header>
      <div
        class="container container-nav"
      >
        <img
          alt="logo"
          class="navbar-logo"
          src="logo.png"
        />
        <nav>
          <ul
            class="navbar-links"
          >
            <li
              class="navbar-links-item"
            >
              <a
                class="item"
                href="/"
              >
                View Properties 
              </a>
            </li>
            <li
              class="navbar-links-item"
            >
              <a
                class="item"
                href="/add-property"
              >
                Add a Property
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </header>
    <div
      class="Properties"
    >
      Properties Page
      <div
        class="property-card"
      >
        <p />
        <p>
          -
        </p>
        <p>
          <svg
            alt="Bedrooms"
            aria-hidden="true"
            class="svg-inline--fa fa-bed icon"
            data-icon="bed"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 640 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M176 288C220.1 288 256 252.1 256 208S220.1 128 176 128S96 163.9 96 208S131.9 288 176 288zM544 128H304C295.2 128 288 135.2 288 144V320H64V48C64 39.16 56.84 32 48 32h-32C7.163 32 0 39.16 0 48v416C0 472.8 7.163 480 16 480h32C56.84 480 64 472.8 64 464V416h512v48c0 8.837 7.163 16 16 16h32c8.837 0 16-7.163 16-16V224C640 170.1 597 128 544 128z"
              fill="currentColor"
            />
          </svg>
        </p>
        <p>
          <svg
            alt="Bathrooms"
            aria-hidden="true"
            class="svg-inline--fa fa-bath icon"
            data-icon="bath"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M32 384c0 28.32 12.49 53.52 32 71.09V496C64 504.8 71.16 512 80 512h32C120.8 512 128 504.8 128 496v-15.1h256V496c0 8.836 7.164 16 16 16h32c8.836 0 16-7.164 16-16v-40.9c19.51-17.57 32-42.77 32-71.09V352H32V384zM496 256H96V77.25C95.97 66.45 111 60.23 118.6 67.88L132.4 81.66C123.6 108.6 129.4 134.5 144.2 153.2C137.9 159.5 137.8 169.8 144 176l11.31 11.31c6.248 6.248 16.38 6.248 22.63 0l105.4-105.4c6.248-6.248 6.248-16.38 0-22.63l-11.31-11.31c-6.248-6.248-16.38-6.248-22.63 0C230.7 33.26 204.7 27.55 177.7 36.41L163.9 22.64C149.5 8.25 129.6 0 109.3 0C66.66 0 32 34.66 32 77.25v178.8L16 256C7.164 256 0 263.2 0 272v32C0 312.8 7.164 320 16 320h480c8.836 0 16-7.164 16-16v-32C512 263.2 504.8 256 496 256z"
              fill="currentColor"
            />
          </svg>
        </p>
        <p>
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-sterling-sign "
            data-icon="sterling-sign"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 320 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M112 223.1H224C241.7 223.1 256 238.3 256 255.1C256 273.7 241.7 287.1 224 287.1H112V332.5C112 361.5 104.1 389.1 89.2 414.9L88.52 416H288C305.7 416 320 430.3 320 448C320 465.7 305.7 480 288 480H32C20.47 480 9.834 473.8 4.154 463.8C-1.527 453.7-1.371 441.4 4.56 431.5L34.32 381.9C43.27 367 48 349.9 48 332.5V288H32C14.33 288 0 273.7 0 256C0 238.3 14.33 224 32 224H48V160.4C48 89.47 105.5 32 176.4 32C190.2 32 203.9 34.22 216.1 38.59L298.1 65.64C314.9 71.23 323.9 89.35 318.4 106.1C312.8 122.9 294.6 131.9 277.9 126.4L196.7 99.3C190.2 97.12 183.3 96 176.4 96C140.8 96 112 124.8 112 160.4V223.1z"
              fill="currentColor"
            />
          </svg>
        </p>
        <p>
          <a
            class="email-link"
            href="mailto:undefined"
          >
             
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-envelope email-icon"
              data-icon="envelope"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M256 352c-16.53 0-33.06-5.422-47.16-16.41L0 173.2V400C0 426.5 21.49 448 48 448h416c26.51 0 48-21.49 48-48V173.2l-208.8 162.5C289.1 346.6 272.5 352 256 352zM16.29 145.3l212.2 165.1c16.19 12.6 38.87 12.6 55.06 0l212.2-165.1C505.1 137.3 512 125 512 112C512 85.49 490.5 64 464 64h-416C21.49 64 0 85.49 0 112C0 125 6.01 137.3 16.29 145.3z"
                fill="currentColor"
              />
            </svg>
            Send email
          </a>
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
